packages/dev/core/src/Physics/physicsHelper.ts

```diff
@@ -1,2 +1,1124 @@
-// ES 6 Compatibility
-export * from "./v1/physicsHelper";
+import type { Nullable } from "../types";

```

Comment:
this file has just moved, right? Just being sure it doesn't need a complete review

I wonder why git hasn't detected that.